tmail-backend/integration-tests/jmap/memory-jmap-integration-tests/src/test/resources/mailetcontainer_openpaas_deployment.xml

```diff
@@ -0,0 +1,179 @@
+<?xml version="1.0"?>
```

Comment:
Why we need a custom mailet container when OpenPaaS?
Is it *really* needed?

Reply:
Hmmm, I think so. We could have one `mailetcontainer.xml` that defines the `OpenPaasAmqpForwardAttribute` but that would lead to server failing when for tests that have OpenPaas disabled.

Reply:
As said in the call we need this because we need a RabbitMQ to push changes to.
We agreed (in a follow up PR)  torun those tests not on memory but on distributed where all distributed tests configures and set up this mailet.